swig.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "swig",
+    "config-opts": [
+        "-with-boost-libs=/app/lib",
+        "--without-alllang",
+        "--with-guile=/app/bin/guile",
+        "--with-guile-config=/app/bin/guile-config"
+    ],
+    "cleanup": ["*"],
+    "sources": [
+        {
+            "type": "archive",
+            "url": "http://prdownloads.sourceforge.net/swig/swig-3.0.12.tar.gz",
+            "sha256": "7cf9f447ae7ed1c51722efc45e7f14418d15d7a1e143ac9f09a668999f4fc94d"
+        }
+    ],
+    "modules": [
+        {
+            "name": "pcre",
```

Comment:
there is a libpcre in the runtime. Isn't it enough?

Reply:
I had just taken that swig.json from another package that was published.  It seems that runtime's pcre works, so I removed that one.